include/podio/ROOTDataSource.h

```diff
@@ -0,0 +1,178 @@
+#ifndef PODIO_DATASOURCE_H__
+#define PODIO_DATASOURCE_H__
+
+// STL
+#include <string>
+#include <vector>
+
+// ROOT
+#include <ROOT/RDataFrame.hxx>
+#include <ROOT/RDataSource.hxx>
+
+// Podio
+#include <podio/CollectionBase.h>
+#include <podio/Frame.h>
+#include <podio/ROOTReader.h>
+
+namespace podio {
+using Record_t = std::vector<void*>;
```

Comment:
Does this have to be so "public", it's a rather general name and unless it's necessary from the ROOT side, this looks like we could move it into a member type of `ROOTDataSource`.

Reply:
using plain `std::vector<void*>` now.